SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/AbstractLockingPolicy.h

```diff
@@ -34,6 +34,7 @@ class SPATIALGDK_API UAbstractLockingPolicy : public UObject
 	virtual ActorLockToken AcquireLock(AActor* Actor, FString LockName = TEXT("")) PURE_VIRTUAL(UAbstractLockingPolicy::AcquireLock, return SpatialConstants::INVALID_ACTOR_LOCK_TOKEN;);
 	virtual bool ReleaseLock(const ActorLockToken Token) PURE_VIRTUAL(UAbstractLockingPolicy::ReleaseLock, return false;);
 	virtual bool IsLocked(const AActor* Actor) const PURE_VIRTUAL(UAbstractLockingPolicy::IsLocked, return false;);
+	virtual void OnOwnerUpdated(AActor* Actor) PURE_VIRTUAL(UAbstractLockingPolicy::OnOwnerUpdated, return;);
```

Comment:
Can this take a const AActor*

Reply:
The argument name in the declaration isn't different from the name in the definitions.